heap/heap.py

```diff
@@ -1,21 +1,78 @@
+# class Heap:
+#   def __init__(self):
+#     self.heapArray = []
+#
+#   def insert(self, value):
+#     pass
+#
+#   def delete(self):
+#     pass
+#
+#   def get_max(self):
+#     pass
+#
+#   def get_size(self):
+#     pass
+#
+#   def _bubble_up(self, index):
+#     pass
+#
+#   def _sift_down(self, index):
+#     pass
+
 class Heap:
   def __init__(self):
-    self.storage = []
+    self.heapArray = [0]
```

Comment:
It looks like you renamed this to something which makes more sense to your brain which I think is GREAT! This will help you learn what is going on in the heap sort more easily.